recipes/qemu-execve/conda_build_config.yaml

```diff
@@ -0,0 +1,28 @@
+c_stdlib_version:
+  - '2.28'                   # [linux and x86_64]
+  - '11.3'                   # [osx and x86_64]
+
+cross_c_stdlib:
+  - sysroot                  # [linux and x86_64]
+  - m2w64-sysroot            # [win and x86_64]
+  - macosx-sdk               # [osx and x86_64]
+cross_c_stdlib_version:
+  - '2.28'                   # [linux and x86_64]
+  - '12'                     # [win and x86_64]
+  - '11.3'                   # [osx and x86_64]
+
+m2w64_c_compiler:            # [win and x86_64]
+  - gcc                      # [win and x86_64]
+m2w64_c_compiler_version:    # [win and x86_64]
+  - '13'                     # [win and x86_64]
+m2w64_c_stdlib:              # [win and x86_64]
+  - m2w64-sysroot            # [win and x86_64]
+m2w64_c_stdlib_version:      # [win and x86_64]
+  - '12'                     # [win and x86_64]
+m2w64_cxx_compiler:          # [win and x86_64]
+  - gxx                      # [win and x86_64]
+m2w64_cxx_compiler_version:  # [win and x86_64]
+  - '13'                     # [win and x86_64]
+
+docker_image:
+  - quay.io/condaforge/linux-anvil-alma-x86_64:8  # [linux and x86_64]
```

Comment:
I think the main issue here is that the infrastructure in staged-recipes is not built to handle image overrides
https://github.com/conda-forge/staged-recipes/blob/6b144d264192503211c8e6deda16fcb5cf90511c/.scripts/run_docker_build.sh#L29-L41

For debugging purposes, you can try overriding:
https://github.com/conda-forge/staged-recipes/blob/6b144d264192503211c8e6deda16fcb5cf90511c/.ci_support/linux64.yaml#L21-L22

The overall situation should improve with https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/6283; until then, we might have to do some special handling for PRs like this.

Reply:
Overriding `linux64.yaml` does not work, but the `linux` part does build locally, so I separated it into another recipe, that I think is ready for review. If it becomes a feedstock, I will continue this PR in the feedstock CI